src/main/java/org/opendatakit/aggregate/form/FormDefinition.java

```diff
@@ -135,9 +136,13 @@ private static final SubmissionAssociationTable getSubmissionAssociation(String
 		    		sa = st;
 		    	}
 		    }
-		}
+			}
 	    return sa;
-	}
+		} catch (Throwable t) {

```

Comment:
`st.getIsPersistenceModelComplete()` in line 125 will throw an `NPE` when that field is `null`. This new `try` block ensures that this exception doesn't scale. Calling code at `FormDefinition` can handle `null` `SubmissionAssociationTable` values.